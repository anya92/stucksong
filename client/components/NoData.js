import React from 'react';

import { InfoContainer } from '../styles/info';

const NoData = () => (
  <InfoContainer>
    <h1>
      You haven't been listening to music lately, have you?
      <span role="img" aria-label="thinking-emoji">ğŸ¤”</span>
    </h1>
    <h2>Please try again later.</h2>
  </InfoContainer>
);

export default NoData;
