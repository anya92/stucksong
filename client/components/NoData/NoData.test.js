import React from 'react';
import { shallow } from 'enzyme';

import NoData from './NoData';

describe('NoData component', () => {
  it('renders NoData component correctly', () => {
    const wrapper = shallow(<NoData />);
    expect(wrapper).toMatchSnapshot();
  });
});
