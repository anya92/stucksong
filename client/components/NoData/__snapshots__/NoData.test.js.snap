// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoData component renders correctly 1`] = `
<glamorous(div)>
  <h1>
    You haven't been listening to music lately, have you?
    <span
      aria-label="thinking-emoji"
      role="img"
    >
      ðŸ¤”
    </span>
  </h1>
  <h2>
    Please try again later.
  </h2>
</glamorous(div)>
`;
